UnitsNet.WindowsRuntimeComponent/UnitsNet.WindowsRuntimeComponent.csproj

```diff
@@ -210,6 +210,9 @@
     <PackageReference Include="JetBrains.Annotations">
       <Version>11.1.0</Version>
     </PackageReference>
+    <PackageReference Include="System.ValueTuple">
```

Comment:
Does this dependency add any extra constraints for consumers?

Reply:
No nuget dependencies listed on these at least. If I understand this nuget correctly, it is a Roslyn package that extends C# language capabilities and should look no different to consumers than `ValueTuple<T1,T2,...>` so if we used this on a public API consumers can still use Item1, Item2 etc. so I think it should be safe.

https://www.nuget.org/packages/System.ValueTuple/
https://blog.marcgravell.com/2017/04/exploring-tuples-as-library-author.html

Reply:
Correct!